.travis.yml

```diff
@@ -1,30 +1,51 @@
-## Documentation: http://docs.travis-ci.com/user/languages/julia/
+# Documentation: http://docs.travis-ci.com/user/languages/julia/
 language: julia
 os:
   - linux
   - osx
+  - windows
 julia:
-  - 0.6
   - 0.7
   - 1.0
+  - 1.1
+  - 1.2
+  - 1.3
   - nightly
 notifications:
   email: false
-git:
-  depth: 99999999
+branches:
+  only:
+  - master
+  - /release-.*/
+  - /v(\d+)\.(\d+)\.(\d+)/
+matrix:
+  allow_failures:
+  - julia: nightly
+codecov: true
 
-## uncomment the following lines to allow failures on nightly julia
-## (tests will run but not make your overall status red)
-#matrix:
-#  allow_failures:
-#  - julia: nightly
-
-after_success:
-  # push coverage results to Coveralls
-  - julia -e '
-    VERSION >= v"0.7.0-DEV.3656" && using Pkg
-    VERSION >= v"0.7.0-DEV.5183" || cd(Pkg.dir("Nullables"))
-    Pkg.add("Coverage")
-    using Coverage
-    Coveralls.submit(Coveralls.process_folder())
-    Codecov.submit(Codecov.process_folder())'
+jobs:
```

Comment:
Is this really needed? Other packages don't list this AFAIK.

Reply:
I do want to make sure we test 32 bit Windows, and that seems to be the way to do that? I've seen a fair number of travis configs that test on Windows, but only 64 bit.

Having said that, I think we're pretty close to getting the GitHub Actions CI story finished, at which I would switch things over to that, so this is probably not a very permanent situation in any case.

Reply:
That sounds quite suboptimal, but well...